stack/.github/workflows/push-image.yml

```diff
@@ -74,15 +101,24 @@ jobs:
         GCR_PASSWORD: ${{ secrets.GCR_PUSH_BOT_JSON_KEY }}
         GCR_PROJECT: "${{ github.repository_owner }}"
       run: |
+
+        if [ "${{ needs.preparation.outputs.push_to_dockerhub }}" == "true" ]; then
+        echo "${DOCKERHUB_PASSWORD}" | sudo skopeo login --username "${DOCKERHUB_USERNAME}" --password-stdin index.docker.io
```

Comment:
Looks like this line is here multiple times. Am I missing something?

Reply:
oops, I fixed it, the indentation was also wrong. I messed it up while committing it. Thanx for your time :) Feel free for any other suggestions, like the format of the registries.json file or if you think we should have that actions splitter into two one for the GCR and one for the dockerhub.